Explorer/Assets/DCL/MarketplaceCredits/MarketplaceCreditsMenuController.cs

```diff
@@ -343,16 +344,17 @@ private void SetSidebarButtonAnimationAsPaused(bool isOn) =>
 
         private void SetSidebarButtonState(CreditsProgramProgressResponse creditsProgramProgressResponse)
         {
-            if (creditsProgramProgressResponse.season.timeLeft <= 0f || creditsProgramProgressResponse.season.isOutOfFunds)
+            if (creditsProgramProgressResponse.IsProgramEnded())
+
             {
                 SetSidebarButtonAnimationAsAlert(false);
                 SetSidebarButtonAsClaimIndicator(false);
                 return;
             }
 
             bool thereIsSomethingToClaim = creditsProgramProgressResponse.SomethingToClaim();
-            SetSidebarButtonAnimationAsAlert(!creditsProgramProgressResponse.HasUserStartedProgram() || !creditsProgramProgressResponse.IsUserEmailVerified() || thereIsSomethingToClaim);
-            SetSidebarButtonAsClaimIndicator(creditsProgramProgressResponse.HasUserStartedProgram() && creditsProgramProgressResponse.IsUserEmailVerified() && thereIsSomethingToClaim);
+            SetSidebarButtonAnimationAsAlert(!creditsProgramProgressResponse.HasUserStartedProgram() || !creditsProgramProgressResponse.IsUserEmailVerified() || (thereIsSomethingToClaim && !creditsProgramProgressResponse.credits.isBlockedForClaiming));
```

Comment:
I understand this is a complicated condition. But for readability, WDYT of something like

```
bool hasNotStarted = !creditsProgramProgressResponse.HasUserStartedProgram();
bool isNotVerified = !creditsProgramProgressResponse.IsUserEmailVerified();
bool canClaim = thereIsSomethingToClaim && !creditsProgramProgressResponse.credits.isBlockedForClaiming;

SetSidebarButtonAnimationAsAlert(hasNotStarted || isNotVerified || canClaim);
```